test/KeyRegistry/KeyRegistry.symbolic.t.sol

```diff
@@ -170,95 +179,7 @@ contract KeyRegistrySymTest is SymTest, Test {
         }
     }
 
-    // Case-splitting tactic: explicitly branching into two states: cond vs !cond
-    function split_cases(bool cond) internal pure {
-        if (cond) return;
-    }
```

Comment:
this is no longer needed in the latest version, thanks to the new address alias reasoning feature.